projects/Mallard/src/hooks/use-config-provider.tsx

```diff
@@ -4,9 +4,10 @@ import DeviceInfo from 'react-native-device-info'
 import { Breakpoints } from 'src/theme/breakpoints'
 import { notificationsEnabledCache } from 'src/helpers/storage'
 import { errorService } from 'src/services/errors'
+import AsyncStorage from '@react-native-community/async-storage'
```

Comment:
Can we use the `src/helpers/storage` instead here? This provides some neat helpers around Async so you dont have to parse it each time.

Reply:
Updated to use helper functions